src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,447 @@
+| DispersionCorrected                 | REQUIRED              | [boolean][]   | Boolean flag specifying whether the blood data have been dispersion-corrected. NOTE: not customary for manual samples, and hence should be set to false.                                                                                                      |
+| WithdrawalRate                      | RECOMMENDED           | [number][]    | The rate at which the blood was withdrawn from the subject. The unit of the specified withdrawal rate should be in mL/s.                                                                                                                                      |
+| TubingType                          | RECOMMENDED           | [string][]    | Description of the type of tubing used, ideally including the material and (internal) diameter.                                                                                                                                                               |
+| TubingLength                        | RECOMMENDED           | [number][]    | The length of the blood tubing, from the subject to the detector in meters.                                                                                                                                                                                   |
+| DispersionConstant                  | RECOMMENDED           | [number][]    | External dispersion time constant resulting from tubing in default unit seconds.                                                                                                                                                                              |
+| Haematocrit                         | RECOMMENDED           | [number][]    | Measured haematocrit, meaning the volume of erythrocytes divided by the volume of whole blood.                                                                                                                                                                |
+| BloodDensity                        | RECOMMENDED           | [number][]    | Measured blood density. Unit of blood density should be in g/mL.                                                                                                                                                                                              |
+
+### Time
+
+If manual blood samples, for example, using a COBRA counter are available,
+then time information as detailed below should be added to the `*_recording-manual_blood.tsv`
+and the values below should be added to the `*_recording-manual_blood.json`.
+If blood samples using an autosampler, for example, using an Allogg autosampler are available,
+then time information as detailed below should be added to the `*_recording-autosampler_blood.tsv`
+and the values below should be added to the `*_recording-autosampler_blood.json`.
```

Comment:
From reading this, there's no difference between autosampler and manual in terms of format. Is there an example of autosampler that would make a distinction clearer?

Reply:
I am not totally sure what you are after, but I have tried to update the text to make it more clear the distinction. 

```suggestion
If manual samples of radioactivity in blood were drawn and measured using e.g. a COBRA counter,
then time information as detailed below should be added to the `*_recording-manual_blood.tsv`
and the values below should be added to the `*_recording-manual_blood.json`.
Instead, if samples of radioactivity were measured automatically in the blood with e.g. an ABSS autosampler (Allogg Technology), then time information as detailed below should be added to the `*_recording-autosampler_blood.tsv`
and the values below should be added to the `*_recording-autosampler_blood.json`. 
```

Reply:
So my question here is just what the point of drawing a strong distinction between manually and autosampled blood data is. I had assumed that with autosampled data, there would be a file-wide T0 offset and a sampling frequency, while manual had to have timestamps. If everything has timestamps, why even bother discussing the two modes, since the data is recorded the same in either case?

Reply:
I can chime in on this one. Despite having time stamps, there can still be a small time offset between manual and automatic samples.  The automatic sampler has a pipe leading to it that the blood runs through, and it can take 20-30 seconds for the blood to reach the sampler (and even then, there's a dispersion effect of blood from its passage through the tube, which we try to correct for).

Another (I think more) important reason to distinguish between these two sets of data are that they come from two different sources.  The automatic samples come from the sampler, and the manual samples come from the chemists who record everything.  The blood from the manual samples is then also used to calculate plasma radioactivity, parent fraction etc. So there are a bunch of different measurements and values all tied to the same sampling timestamps, from the same manual blood samples, which are measured in different ways.

There was a suggestion to have them all together before, but the resulting table was extremely sparse because there are so many automatic samples and so few manual samples, where all the extra columns were tied to the manual samples. It was decided that this would likely result in issues with data entry, through blending data from the different sources, as well as making it much more difficult to look through the data due to all the NAs everywhere.

Reply:
I'm not suggesting people combine their recordings unwisely. Presumably there will often be different metadata that apply to each method, fully justifying multiple files. The point is simply that there is no difference in the data type, just the contents and metadata. I will make suggestions in #730 that should help clarify further.